item2/Dictionary.java

```diff
@@ -0,0 +1,53 @@
+import java.util.*;
+
+public class Dictionary {
+  private Map<String, Object> dict = new HashMap<String, Object>();
```

Comment:
nit: elements might be a good variable name

Reply:
updated in latest commit.